util/chunk/chunk.go

```diff
@@ -40,6 +47,12 @@ type Chunk struct {
 	requiredRows int
 }
 
+func (c *Chunk) assertNilSel() {
```

Comment:
Better not to use this function.

Reply:
It's difficult to make all existed functions be compatible with `Sel` in a single PR, so I use it to ensure there is no `Sel` when these incompatible functions are called.

I will make all functions be compatible with `Sel` when refactoring executors in the next few PRs and finally remove `assertNilSel`.

Or do you have a better way to avoid this incompatible problem?